tests/periph_gpio/Makefile

```
@@ -7,7 +7,19 @@ USEMODULE += shell
 USEMODULE += shell_commands
 USEMODULE += benchmark
 
+BOARDS_BENCH_PORT_1 = \
+    slstk3402a \
+    #
+
+ifneq (,$(filter $(BOARDS_BENCH_PORT_1),$(BOARD)))
```

```suggestion
ifneq (,$(filter $(BOARD),$(BOARDS_BENCH_PORT_1)))
```
would look better IMHO